doc/manual/manual.md

```diff
@@ -87,14 +92,22 @@ SAW also uses several environment variables for configuration:
   ~ Specify a colon-delimited list of directory paths to search for Cryptol
   imports (including the Cryptol prelude).
 
+`PATH`
+
+  ~ If the `--java-bin-dirs` option is not set, then the `PATH` will be
+    searched to find a Java executable.
+
 `SAW_IMPORT_PATH`
 
   ~ Specify a colon-delimited list of directory paths to search for imports.
 
 `SAW_JDK_JAR`
 
   ~ Specify the path of the `.jar` file containing the core Java
-  libraries.
+  libraries. TODO RGS: What should we do about this? It doesn't really serve a
```

Comment:
I've left a TODO here since I'm really not sure what we should do with the `SAW_JDK_JAR` environment variable. It doesn't do much now that `--java-bin-dirs`/`PATH` are consulted. We might envision `SAW_JDK_JAR` as just an environment variable version of `--jars`, but that's not how it's actually implemented currently—it just takes a single `.jar` as an argument. (Contrast this with `SAW_IMPORT_PATH`, which takes a list of paths as an argument.) Moreover, the name "`SAW_JDK_JAR`" seems to suggest that it's only used for one specific `.jar` file.

What should we do about `SAW_JDK_JAR`? Deprecate it? Remove it? Repurpose it? Something else?

Reply:
Although `SAW_JDK_JAR`'s role is much less clear in a post-`--java-bin-dirs` world, I've decided to just leave it, as it's not actively harming anything. We can revisit whether we want to do something with it later.